mhubert-pmn/Front-Url-Shortener
# [Feature] Create header

## Task
Create header

## Type of change

## Description
- Create Header, SubHeader, Menu and MobileMenu components
- Add condition to display links and buttons

**/!\ The buttons are unstyled. They will be replaced by the Button component later**
**/!\ The Header component will be placed into the Layout component to be displayed on every view of the app** 

## Testing scenario
In the terminal:
- Launch and build: `docker-compose up --build
- Go to [http://localhost:3000/](http://localhost:3000/) and check if the header appears
- Change the size of your screen to see the responsive menu

## Trello card
[https://trello.com/c/ofOEWHf0](https://trello.com/c/ofOEWHf0)
